CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Refactor ParseError to enrich Sentry with context and to inquire about Sentry errors like #4096  [#218](https://github.com/datagouv/hydra/pull/218)
 - Remove legacy routes [#203](https://github.com/datagouv/hydra/pull/203)
+- Handle Sentry issue #2333 with more explicit error reporting and not raising an error anymore for uadata 404 errors [#213](https://github.com/datagouv/hydra/pull/213)
```

Comment:
I think error #2333 will not exist anymore in the future, could you describe it shortly instead?
```suggestion
- Handle Sentry issue #2333 with more explicit error reporting and not raising an error anymore for udata 404 errors [#213](https://github.com/datagouv/hydra/pull/213)
```

Reply:
Makes sense. Done!